back/spec/acceptance/ideas_spec.rb

```diff
@@ -1381,6 +1396,36 @@
     end
   end
 
+  context 'when admin' do
+    let(:user) { create(:admin) }
+
+    before do
+      header_token_for user
+    end
+
+    get 'web_api/v1/ideas' do
+      parameter :phase, 'Filter by project phase', required: false
+
+      example 'List all ideas in a phase of a project - ideas_phase basket count is returned' do
+        pr = create(:project_with_active_budgeting_phase)
+        phase = pr.phases.first
+        ideas = create_list(:idea, 2, phases: [phase], project: pr)
+        basket = create(:basket, participation_context: phase, submitted_at: nil)
+        basket.update!(ideas: ideas, submitted_at: Time.zone.now)
+        SideFxBasketService.new.after_update basket, user
+
+        do_request phase: phase.id
+        assert_status 200
+        expect(response_data.size).to eq 2
+        expect(response_data.pluck(:id)).to match_array [ideas[0].id, ideas[1].id]
+        ideas_phases = json_response_body[:included].map { |i| i if i[:type] == 'ideas_phase' }.compact!
+        expect(ideas_phases.size).to eq 2
+        expect(ideas_phases[0][:attributes][:baskets_count]).to eq 1
```

Comment:
Given that this test should always return 1 is it OK to leave as it is for now? Not entirely sure how to change it

Reply:
Yes, fine to leave it. Otherwise, you could try something like `expect(ideas_phases.map{ |ideas_phase| ideas_phase.dig(:attributes, :baskets_count) }).to contain_exactly 1, 1`, so that order does not matter.